client/package.json

```diff
@@ -3,24 +3,25 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/core": "^7.12.7",
```

Comment:
You could install babel/core and preset-env as developer dependency, as they are mainly needed for development.